### PR TITLE
Add tests for upload_sample_input

### PR DESCRIPTION
**Pull request by owenl131**

### CODE REVIEW

**Comment by owenl131** [comment]
@AvaisP In views.py, in the upload_sample_input function line 442, data.iteritems has to be changed to data.items as python 3 no longer supports iteritems https://stackoverflow.com/questions/30418481/error-dict-object-has-no-attribute-iteritems-when-trying-to-use-networkx

**Comment by AvaisP** [comment]
Please fix the conflicts :smile:

File: api/views.py [hunk 1]
```diff
@@ -439,7 +439,7 @@ def upload_sample_input(request):
     data = request.data
     demo_id = data["demo_id"]
     demo = Demo.objects.get(id=demo_id)
-    for key, value in data.iteritems():
+    for key, value in data.items():
```
Comment (AvaisP) [inline api/views.py]: Why?

Comment (owenl131) [reply api/views.py]: python 3 no longer supports iteritems, as I mentioned in the comment above.

**Comment by owenl131** [comment]
Ok, fixed the conflicts

File: api/views.py (hunk 1 above)
Comment (AvaisP) [reply api/views.py]: You'll need to return iter(data.items()) right? https://docs.python.org/3.3/library/stdtypes.html#dictionary-view-objects

Comment (owenl131) [reply api/views.py]: I think the for .. in loop implicitly uses an iterator

Comment (AvaisP) [reply api/views.py]: :+1:

**Pull request merged**